src/morphocut/annotation.py

```diff
@@ -36,23 +55,41 @@ def transform(self, image, mask):
         # Draw contours
         boundary_mask = skimage.segmentation.find_boundaries(dilated_mask, mode="outer")
 
-        contour_image = img_as_ubyte(image, True)
-        color = (0, 255, 0)
+        contour_image = skimage.color.gray2rgb(img_as_ubyte(image, True))
+        #color = (0, 255, 0)
 
-        contour_image[boundary_mask] = color
+        contour_image[boundary_mask] = self.color
 
         return contour_image
 
 @ReturnOutputs
 @Output("parent_contour")
 class DrawContoursOnParent(Node):
+    """
+    Dilate the image and make boundary around the dilated image and outline pixels in the background around 
+    object boundaries.
+
+    .. note::
+        Uses the skimage library :py:func:`skimage.morphology.binary_dilation` and :py:func:`skimage.segmentation.find_boundaries`.
+
+    Args:
+        image (np.ndarray or Variable[np.ndarray]): An image to be contoured.
+        mask (np.ndarray or Variable[np.ndarray]): Mask of an image.
+        color (tuple): Value of RGB colors as a tuple.
+        dilate_rel (int): Relative Dilation value.
+        dilate_abs (int): Absolute Dilation value.
+
+    Returns:
+        Variable[np.ndarray]: Contoured image.
+    """
 
-    def __init__(self, image, mask, output, parent_slice, dilate_rel=0, dilate_abs=0):
+    def __init__(self, image, mask, parent_slice, color, dilate_rel=0, dilate_abs=0):
```

Comment:
Document `parent_slice: Slice that locates the child object in the parent image.`

Rename `image` -> `parent_image`, `mask` -> `child_mask` for clarity.

Restore `output` -> `ref_parent_image_out (np.ndarray pr Variable): Image onto which the object will be drawn. This is passed by reference and will be altered in-place.` as a parameter. Remove image as a parameter.

```
.. warning::
        `parent_image_out` will be altered in-place. Pass a copy of the original image.
```